static/i18n/en/app.json

```diff
@@ -696,6 +696,7 @@
   "tron": {
     "voting": {
       "manageTP": "Manage Tron Power",
+      "warnEarnRewards": "You need at least 1TRX to start earning rewards",
```

Comment:
this must not be in the translation. it should be `You need at least {{amount}} to start earning rewards` and there is a missing space (and also I think it's `TRX 1`)